src/wopiutils.py

```diff
@@ -19,14 +20,25 @@
 # Convenience dictionary to store some context and avoid globals
 _ctx = {}
 
-credentials = {}
 
+class ViewMode(IntEnum):
```

Comment:
Why do you use intEnum instead of Enum?

Reply:
Very good question: python does not support ints in `Enum` types, and one has to use this derived type instead!

Reply:
Yes, but why do we use ints instead of "INVALID"= "INVALID", "VIEW_ONLY"="VIEW_ONLY" etc?

Reply:
I thought that the CS3API enum itself was defined with ints, and Reva would pass ints to the call. Can we have strings instead? if so, I'm happy to use them :-)

Reply:
I guess it depends on how we call the wopi server from reva. I don't think passing a string enum should be a problem.

Reply:
Well, happy to go for that then - and there's no eventuality of "nil" or invalid values as those would be rejected by the open call itself.
I'll change the `Enum` to match strings.